docs/client_reference.rst

```diff
@@ -1561,6 +1568,21 @@ All exceptions are available as members of *aiohttp* module.
 
       Invalid URL, :class:`yarl.URL` instance.
 
+.. class:: ContentDisposition
+
+    Represent Content-Disposition header
+
+    .. attribute:: value
+
+    Value of Content-Disposition header itself, e.g. ``attachment``.
+
+    .. attribute:: filename
+
+    Content filename extracted from parameters. May be ``None``.
+
+    .. attribute:: parameters
+
+    A :class:`dict` instance contains all parameters.
```

Comment:
Actually, it's not a dict, but `MapptingProxy`. Difference in mutability.

Reply:
We could just replace it with `*mapping*` for sake of simplicity.
I pretty sure not many python developers know what `MappingProxy` is :)

Reply:
Oops

Reply:
Good idea.